principles.md

```diff
@@ -3,49 +3,42 @@
 Define a unit of software delivery called a Standard Container.
 The goal of a Standard Container is to encapsulate a software component and all its dependencies in a format that is self-describing and portable, so that any compliant runtime can run it without extra dependencies, regardless of the underlying machine and the contents of the container.
 
-The specification for Standard Containers is straightforward.
-It mostly defines 1) a file format, 2) a set of standard operations, and 3) an execution environment.
+The specification for Standard Containers defines:
 
-A great analogy for this is the shipping container.
-Just like how Standard Containers are a fundamental unit of software delivery, shipping containers are a fundamental unit of physical delivery.
+1. configuration file formats
+2. a set of standard operations
```

Comment:
well, this round is more about clean-up - with just a touch of clarity.  The entire 'standard operations' thing is still a bit of a mystery to me since I don't know how you define an operation w/o defining something like an interface or API  :-)